records_mover/records/cli.py

```diff
@@ -82,14 +88,18 @@ def main() -> None:
     # https://stackoverflow.com/questions/15405636/pythons-argparse-to-show-programs-version-with-prog-and-version-string-formatt
     parser.add_argument('-V', '--version', action='version', version="%(prog)s ("+__version__+")")
     subparsers = parser.add_subparsers(help='subcommand_help')
+    from records_mover import Session
+    bootstrap_session = Session()
```

Comment:
We're doing more work now during Session initialization by fetching GCP credentials - which, combined with the annoying logs suppressed above, made doing this in a tight loop pretty apparent.  Instead, let's do this once and pass it in.